codespell_lib/data/dictionary.txt

```diff
@@ -6705,7 +6705,7 @@ considderation->consideration
 considdered->considered
 considdering->considering
 considerd->considered
-considere->considered
+considere->considered, consider,
```

Comment:
As suggested in https://github.com/codespell-project/codespell/pull/1881#issuecomment-787955872:

```suggestion
considere->consider, considered,
```